apps/api/.github/workflows/playwright.yml

```
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```

This will enable this workflow to run on GitHub actions right away after the merge to master, we don't want that yet, you'll do that in a separate milestone.

So for the time being we can avoid running this workflow automatically by using `workflow_call`, this will also allow you to manually run it on actions if you need. 

```suggestion
on:
  workflow_call:
```

---

I will make that change, thanks!